learn/index.md

```diff
@@ -2,7 +2,32 @@
 title: Learn About Tanssi Infrastructure Protocol
 description: Learn about Tanssi protocol and how it helps developers to build and deploy appchains by handling infrastructure complexities and providing key integrations.
 template: index-page.html
-hide:
- - toc
- - feedback
 ---
+
+# Learn
+
+Launching an appchain has traditionally been a challenging and time-consuming process, requiring expertise in infrastructure, security, and operational setup. Tanssi simplifies this journey by removing the complexities and making appchain deployment fast and accessible. With Tanssi, developers can rely on seamless block production, integration with essential tools, and the flexibility to choose security providers that best fit their blockchain environment, empowering them to focus on innovation rather than technical hurdles.
+
+This section dives into the core elements that define Tanssi as a leading solution for appchain development. From its modular framework and runtime customization to transaction management and interoperability, explore how Tanssi transforms appchain development into an accessible and efficient process, drastically reducing time-to-market while ensuring high performance and scalability.
+
+Whether you’re starting your journey into appchains or deepening your understanding of Tanssi’s unique capabilities, these documents provide everything you need to understand how Tanssi sets a new standard for appchain infrastructure.
+
+
+## Explore What This Section Covers
+
+:::INSERT_GENERATED_CARDS:::
+
+
+## Want to jump right into the action? 
```

Comment:
```suggestion
## Want to Jump Right into the Action?
```

For titles, we use title case that follows the Chicago case style. If I am ever in doubt, and sometimes when I'm just lazy, I use this website: https://titlecaseconverter.com/